tileserver/__init__.py

```diff
@@ -234,9 +234,10 @@ def async_store(store, tile_data, coord, format):
 def async_update_tiles_of_interest(redis_cache_index, coord):
     """update tiles of interest set"""
     try:
-        redis_cache_index.index_coord(coord)
+        if coord.zoom <= 20:
+            redis_cache_index.index_coord(coord)
         if coord.zoom > 18:
-            coord_at_z18 = coord.zoomTo(18)
+            coord_at_z18 = coord.zoomTo(18).container()
```

Comment:
I'm not sure I follow the logic here: If a tile is at zoom 18 or lower, then the first branch is executed. At zoom 21 or higher the second branch is executed. At zooms 19 and 20, _both_ will be executed - won't this double the count at those zooms?